addons-cluster/etcd/templates/cluster.yaml

```diff
@@ -15,10 +15,10 @@ spec:
   {{- end }}
   componentSpecs:
     - name: etcd
-      {{- if .Values.peerService.name }}
+      {{- if .Values.peerService.enabled }}
       services:
-        - name: {{ .Values.peerService.name }}
-          serviceType: {{ .Values.peerService.type }}
+        - name: peer
+          serviceType: LoadBalancer
```

Comment:
why LB is the only choice ?

Reply:
cause default peerService in etcd is headless svc, and we can specify LoadBalancer svc to introduce ip addrs to enable multi-cluster communication.
so this field is only used for multi-cluster etcd, and I haven't test for NodePort svc for multi-cluster, not sure if it can run well.

Reply:
sounds reasonable, but NodePort service is not supported for multi-cluster yet, here is [related issue](https://github.com/apecloud/kubeblocks/issues/7916).

Reply:
anyway, I make it configurable for future implementation